package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@girder/components",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
I've already released this by hand while testing token auth, so it needs to be merged.